### DISCUSSION TITLE
Homepage url is missing protocol

### DISCUSSION THREAD

**Discussion by sa-** [alpinejs/alpine#4376] [7. Bugs (archived)] [answered]
<img width="816" alt="image" src="https://github.com/user-attachments/assets/2c0decbf-c8d6-4483-a27a-adc014201412">

**Comment by SimoTod** [comment, marked as answer]
In reality (although it would be better not to suggest it), that is a valid syntax to say "use whatever protocol your page is loaded with" (No protocol-relative URL).

That page is more of an introductory splash screen, a better CDN to use is the one mentioned in  https://alpinejs.dev/start-here
